stack.ghc-8.4.yaml

```diff
@@ -1,66 +0,0 @@
-flags: {}
```

Comment:
I think I might like to keep the Stack configuration files for just a little while, mostly because we don't have a good coverage analysis approach without using Stack.

Reply:
Makes sense. I'll keep them in until we have something equivalent without them.